tmt-app/src/Components/NavBar.jsx

```diff
@@ -0,0 +1,171 @@
+import CloseIcon from '@mui/icons-material/Close';
+import MenuIcon from '@mui/icons-material/Menu';
+import {
+  AppBar,
+  Box,
+  Container,
+  Drawer,
+  IconButton,
+  Link,
+  List,
+  ListItem,
+  ListItemButton,
+  Toolbar,
+  Typography,
+} from '@mui/material';
+import React, { useState } from 'react';
+import { Link as RouterLink } from 'react-router-dom';
+
+import BTLogo from '../Images/icons/BT_Logo_1.svg';
+
+function NavBar() {
+  let [navMenu, setNavMenu] = useState(false);
+
+  const handleOpenNavMenu = () => {
+    setNavMenu(true);
+  };
+  const handleCloseNavMenu = () => {
+    setNavMenu(false);
+  };
+  const tabs = [
+    {
+      name: 'Majors & Careers',
+      path: '/',
+    },
+    {
+      name: 'Team',
+      path: '/team',
+    },
+    {
+      name: 'Take the Quiz',
+      path: '/quiz/classes',
+    },
+  ];
+  return (
+    <AppBar
+      sx={{
+        position: 'sticky',
+        backgroundColor: 'BTMilkTea.main',
+        marginBottom: '3.5rem',
+      }}
+    >
+      <Container maxWidth="xl">
+        <Toolbar sx={{ justifyContent: 'space-between' }}>
+          <Link key="Homepage" to="/" component={RouterLink}>
+            <img src={BTLogo} alt="logo" />
+          </Link>
+          <Box
+            sx={{
+              display: {
+                xs: 'flex',
+                md: 'none',
+              },
+            }}
+          >
+            <IconButton
+              aria-label="options"
+              color="black"
+              onClick={handleOpenNavMenu}
+            >
+              <MenuIcon />
+            </IconButton>
+            <Drawer
+              anchor="right"
+              open={navMenu}
+              onClose={handleCloseNavMenu}
+              ModalProps={{ disableScrollLock: 'true' }}
+              sx={{
+                display: {
+                  xs: 'flex',
+                  md: 'none',
+                },
+              }}
+            >
+              <Box>
+                <ListItem sx={{ justifyContent: 'flex-end' }}>
+                  <IconButton
+                    aria-label="options"
+                    color="black"
+                    onClick={handleCloseNavMenu}
+                  >
+                    <CloseIcon />
+                  </IconButton>
+                </ListItem>
+                {tabs.map((tab, id) => {
```

Comment:
map's 2nd param is `index` btw, we should use that over `id` to not get confused

Reply:
sounds good! I will make the change